crates/swc_ecma_quote/README.md

```diff
@@ -0,0 +1,3 @@
+# `swc_core` replaces `swc_ecma_quote`
+
+Instead of directly importing `swc_ecma_quote`, `swc_core` provides equivalent `swc_core::quote` and `swc_core::quote_expr`.
```

Comment:
We can remove this in next version I think...
So the README for swc_ecma_quote show proper instruction

Reply:
Yes, that was the reason I added readme but didn't remove whole pkgs yet.